README.md

```diff
@@ -1,9 +1,9 @@
 Kazoo
 =====
 
-![Travis Build](https://travis-ci.org/python-zk/kazoo.svg?branch=master)
+[![Build Status](https://travis-ci.org/python-zk/kazoo.svg?branch=master)](https://travis-ci.org/python-zk/kazoo)
 
-![Latest Version](https://img.shields.io/pypi/v/kazoo.svg)
+[![PyPI version](https://badge.fury.io/py/kazoo.svg)](https://badge.fury.io/py/kazoo)
```

Comment:
I don't understand this last one:

1. Why switch away from `shields.io` as the image source?
2. Why is the URL not pointing directly to PyPI? https://pypi.org/project/kazoo/

Reply:
1. Didn't realize that it was a badge service (thought it was maybe just a static image host or something) until you mentioned it.
2. `badge.fury.io` automatically redirects to PyPi kazoo. It's just the default way to use them as mentioned in their docs.

I've added a commit to switch back to shields.io and just fix the link instead after reading your comment though.